src/main/java/it/academy/app/shared/ErrorMessages.java

```diff
@@ -0,0 +1,54 @@
+package it.academy.app.shared;
```

Comment:
Package name doesn't look right. What does shared imply?
Maybe keep it inside exception or validation or similar?